rules/windows/powershell/powershell_data_compressed.yml

```diff
@@ -1,11 +1,11 @@
-title: Data Compressed - Powershell
+title: Data Compressed - PowerShell
 id: 6dc5d284-69ea-42cf-9311-fb1c3932a69a
 status: experimental
-description: An adversary may compress data (e.g., sensitive documents) that is collected prior to exfiltration in order to make it portable and minimize the amount of data sent over the network
+description: An adversary may compress data (e.g., sensitive documents) that is collected prior to exfiltration in order to make it portable and minimize the amount of data sent over the network.
 author: Timur Zinniatullin, oscd.community
 date: 2019/10/21
 references:
-    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1002/T1002.yaml
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1560/T1560.yaml
```

Comment:
more this reference :  https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1560/T1560.md

Reply:
I just did a quick search for references to the atomic-red-team repository in the entire Sigma repository and found that it was a mix of `.yaml` and `.md` links (though there appeared to be more ending with `.md`).

I chose `.yaml` because each rule's `.md` version is only periodically generated automatically (i.e. any changes are first made to the `.yaml` version).

But either is fine with me. :)

Reply:
It's just that .md is more user-friendly for those new to security.
For me yaml is my second language before English :relaxed:

Reply:
Yes, please use .md

Reply:
Awesome - thank you for clarifying your positions on this!

My latest commit did the change.